cities/cities.json

```diff
@@ -88,8 +88,8 @@
         "label": "Frankfurt am Main"
     },
     "friedrichsdorf": {
-	"id": "friedrichsdorf",
-	"label": "Friedrichsdorf"
+        "id": "friedrichsdorf",
```

Comment:
Es ist guter Stil nur die Stellen im Code zu ändern, die für die eigene Änderung notwendig sind :stuck_out_tongue_winking_eye:

Reply:
Sorry, auto format :) Ich reverte meine Änderungen einfach :)

Reply:
Thank you.